index.html

```diff
@@ -206,9 +206,9 @@ <h2 class="primary-text">About me</h2>
 
       <form class="form" method="post" action="https://formspree.io/f/xnqrekyo">
         <label for="name"></label>
-        <input type="text" class="single input-field" id="name" name="name" maxlength="30" placeholder="Full name" required>
+        <input type="text" class="single input-field userData" id="name" name="name" maxlength="30" placeholder="Full name" required>
         <label for="email"></label>
-        <input type="email" class="single input-field" id="email" name="email" placeholder="Email address" required>
+        <input type="email" class="single input-field userData" id="email" name="email" placeholder="Email address" required>
         <label for="message"></label>
         <textarea class="input-field" id="message" name="message" rows="3" maxlength="500" placeholder="Enter text here" required></textarea>
         <input class="see-project" id="submit" type="submit" value="Get in touch">        
```

Comment:
Nice implementation so far. :+1: However, the data for all the three inputs including the textarea should be populated and stored in the local storage, but I see the implementation on only the name and email inputs. Kindly add the textarea input. :wink: